tests/resources/kafka/Makefile

```diff
@@ -0,0 +1,9 @@
+IMAGE=quay.io/opendatahub-ci/kafka-test:latest
+WITH_BUILDER="docker"
```

Comment:
was docker the intent here or should it have been podman?

Reply:
So, this is "wrong" overall since s2i does not support `--with-builder` in current release - I have a custom build from a pending PR installed...so it should basically be "nothing" by default (which defaults to docker really). I changed it to work with  released version of s2i